vllm/model_executor/guided_decoding.py

```diff
@@ -9,15 +9,42 @@
 from pydantic import BaseModel
 
 from vllm.entrypoints.openai.protocol import CompletionRequest, ChatCompletionRequest
-from vllm.model_executor.guided_logits_processors import JSONLogitsProcessor, RegexLogitsProcessor
+from vllm.model_executor.guided_logits_processors import JSONLogitsProcessor, RegexLogitsProcessor, CFGLogitsProcessor
 
 
 class GuidedDecodingMode(Enum):
     JSON = "json"
     REGEX = "regex"
     CHOICE = "choice"
+    CFG = "cfg"
```

Comment:
A dumb question: does `CFG` mean `config`? only for grammar config?

Reply:
Context free grammar, good point that I should probably change it to `grammar` to disambiguate